helix-core/src/main/java/org/apache/helix/model/ParticipantHistory.java

```diff
@@ -40,7 +41,9 @@
   private static Logger LOG = LoggerFactory.getLogger(ParticipantHistory.class);
 
   private final static int HISTORY_SIZE = 20;
-  private enum ConfigProperty {
+  final static String HISTORY_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss:SSS";
```

Comment:
private?

Reply:
If required in the test, then either put it in HelixConstants or redefine a string in the test case (this helps to prevent if anyone changes the format that breaks the compatibility)

Reply:
That's a great idea; I'll just redefine it in test.